docs/user/vsphere/requirements.md

```diff
@@ -0,0 +1,17 @@
+# Resource Requirements
```

Comment:
vSphere IPI requires DHCP

Reply:
I haven't reviewed the keepalived config recently. But, we added an issue multicast VRRP creating issues with multiple clusters on the same broadcast domain. https://issues.redhat.com/browse/KNIDEPLOY-1978

Reply:
I updated. Please take a look if more detail should be provided or if this sounds wrong.